core/LocOffsets.h

```diff
@@ -51,6 +51,10 @@ struct LocOffsets {
 };
 CheckSize(LocOffsets, 8, 4);
 
+template <typename H> H AbslHashValue(H h, const LocOffsets &m) {
```

Comment:
Is this function used when inserting into `UnorderedSet`?

Reply:
yep